gemma-core/src/main/resources/ubic/gemma/model/analysis/Investigation.hbm.xml

```diff
@@ -130,6 +130,13 @@
 					<!-- cannot be non-null because subsets and generic experiments don't have curation details -->
 					<column name="CURATION_DETAILS_FK" not-null="false" sql-type="BIGINT" unique="true"/>
 				</many-to-one>
+				<set name="singleCellExpressionDataVectors" lazy="true" fetch="select" inverse="true"
+					 cascade="all-delete-orphan">
```

Comment:
We should remove the `-delete-orphan` and manage vectors the same way we do for raw and processed ones.

Reply:
That include bulk insertion, removal, etc.